<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="tab_gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Divider</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QPushButton" name="Div_apply_btn">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QScrollArea" name="DivTab_scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>263</width>
              <height>114</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="vl_sublayout"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="Div_add_name_btn">
           <property name="text">
            <string>Type new name:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Float</string>
        </attribute>
        <widget class="QPushButton" name="flt_add_name_btn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>131</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Type new name:</string>
         </property>
        </widget>
        <widget class="QScrollArea" name="fltTab_scrollArea">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>273</width>
           <height>102</height>
          </rect>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>271</width>
            <height>100</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="flt_vl_sublayout"/>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QLabel" name="flt_val_lbl">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Float Value:</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="flt_Unlimited_radioBtn">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>10</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>limited</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="flt_Min_spnBox">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>30</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="flt_Max_spnBox">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>30</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="flt_Min_lbl">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>10</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Min</string>
         </property>
        </widget>
        <widget class="QLabel" name="flt_Max_lbl">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>10</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Max</string>
         </property>
        </widget>
        <widget class="Line" name="flt_horiz_line">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="Line" name="flt_vert_line">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>10</y>
           <width>20</width>
           <height>51</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="flt_apply_btn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>271</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Enum Switch</string>
        </attribute>
        <widget class="QScrollArea" name="enmTab_scrollArea_">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>273</width>
           <height>102</height>
          </rect>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>271</width>
            <height>100</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="enm_vl_sublayout"/>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="enm_masterName_lbl">
            <property name="text">
             <string>Master Name: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="enm_masterName_line"/>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="enm_apply_btn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>160</y>
           <width>271</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>10</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QPushButton" name="enm_add_name_btn">
            <property name="text">
             <string>Type new name:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
